### ISSUE TITLE
Broken Image Links - Testing Round 7 in Prod

### ISSUE THREAD

**Issue by emjohnst** [bcgov/BCHeritage#1034]
Saved Searches - broken image link when creating and uploading image to saved searches

Details page - Broken image link for Fisgard lighthouse - Fisgard Lighthouse - DcRu-152

![Image](https://github.com/user-attachments/assets/52d0c9f9-f3b7-4ad2-bbff-de347f3f9012)

**Comment by bferguso** [comment]
The missing DcRu-152 link was caused by a bug that has been in the system since 1.0.0, but was not noticed as it displayed correctly in the form, but doesn't on the details page. The file URL for these starts with '/files/..' rather than '/bcrhp/files'. We'll create a migration to fix that.

The issue for the saved searches is the opposite - the file URLs start with '/bcrhp/bcrhp/files'. Need to track down why that is. In the last migration we changed the media prefix from `/files/` to `/bcrhp/files/` to fix the first issue.
